x-pack/agent/docs/agent_configuration_example.yml

```diff
@@ -211,23 +228,23 @@ datasources:
         # buckets?
         - type: s3
           credential_profile_name: fb-aws
+          #shared_credential_file: /etc/filebeat/aws_credentials
+          queue_url: https://sqs.myregion.amazonaws.com/123456/sqs-queue
```

Comment:
I'm still confused on how this will work to pick the right logs / files per stream.

Reply:
OK, I've dug again in the code, I think its not currently possible to have a single SQS for all the kind of AWS logs that we are currently handling, each ingest pipeline are different in every fileset.

So a different queue is indeed needed, I twould be nice to have a single queue and to funnel the processing to the right modules. But that would require changes on the module definition and this would be harder to "opt-in" to a single fileset.

Reverting the change for having queue **per stream definition.**

Reply:
Thanks for digging into this @ph . @kaiyan-sheng Perhaps the AWS filebeat module docs could need some details on this.